### PR TITLE
formio validations features enabled and also reflected in form as val…

### PR DESCRIPTION
**Pull request by suvarnakale**
…idations

### CODE REVIEW

File: src/app/issure/advance-editor/advance-editor-config.ts [hunk 1]
```diff
@@ -150,15 +150,6 @@ var editTextfieldCompValidation = [
         "key": "validateOn",
         "ignore": true
     },
```
Comment (Pratikshakhandagale) [inline src/app/issure/advance-editor/advance-editor-config.ts:152]: @suvarnakale is min and max length handled for number too?

File: src/app/issure/advance-editor/advance-editor.component.ts [hunk 2]
```diff
@@ -521,12 +533,22 @@ export class AdvanceEditorComponent implements OnInit {
       tempFjson[fieldObj.key]['description'] = fieldObj.description
     }
 
+    if(fieldObj.hasOwnProperty('validate') && fieldObj.validate.minLength){
```
Comment (Pratikshakhandagale) [inline src/app/issure/advance-editor/advance-editor.component.ts]: @suvarnakale same code repeated here, create one fuction like below
checkValidate(tempjson, element, validateType) .
validateType will be **max, min or customMessage**

File: src/app/issure/advance-editor/advance-editor-config.ts (hunk 1 above)
Comment (suvarnakale) [reply src/app/issure/advance-editor/advance-editor-config.ts:152]: @Pratikshakhandagale Please check the ticket, I have described how min and max validation will be handled for number.

File: src/app/issure/advance-editor/advance-editor.component.ts (hunk 2 above)
Comment (suvarnakale) [reply src/app/issure/advance-editor/advance-editor.component.ts]: okay

**Review by Pratikshakhandagale — APPROVED** [review]

**Pull request merged**